### DISCUSSION TITLE
get tx/rx speed of outbounds

### DISCUSSION THREAD

**Discussion by mokhtarabadi** [eycorsican/leaf#69] [Ideas]
it's great if you can add an API server like v2ray to control some of the leaf features and also get the real-time speed (download/upload bytes per second) for each outbound connections, or can you help me to get rx/tx speed I can transfer that by using UNIX socket (like shadowsocks android)

**Comment by mokhtarabadi** [comment]
@eycorsican 
how stat outbound works?
I try to use it but seems didn't work

**Reply by eycorsican** [reply]
The unfinished stat outbound is very likely to be removed in the future and I would like a simpler solution for traffic statistics, but I don't have schedule for this feature.

**Reply by mokhtarabadi** [reply]
Do you have any plan for this feature?

**Reply by eycorsican** [reply]
No updates 🙄

**Reply by eycorsican** [reply]
It's a wanted feature.
